auomsctl.cpp

```diff
@@ -596,21 +582,9 @@ void enable_service() {
     std::string path;
     std::vector<std::string> args;
 
-    if (PathExists(SYSTEMCTL_PATH)) {
-        path = SYSTEMCTL_PATH;
-        args.emplace_back("enable");
-        args.emplace_back(SYSTEMD_SERVICE_FILE);
-    } else if (PathExists(CHKCONFIG_PATH)) {
-        path = CHKCONFIG_PATH;
-        args.emplace_back("--add");
-        args.emplace_back(AUOMS_SERVICE_NAME);
-    } else if (PathExists(UPDATE_RC_PATH)) {
-        path = UPDATE_RC_PATH;
-        args.emplace_back(AUOMS_SERVICE_NAME);
-        args.emplace_back("defaults");
-    } else {
-        throw std::runtime_error("Failed to locate service control utility");
-    }
+    path = SYSTEMCTL_PATH;
```

Comment:
In the old if this path does not exist, it is throwing std::runtime_error("Failed to locate service control utility");

May i know why this check is eliminated here?

Reply:
If the systemctl utility is not present, then the command will fail with a "path not found" error. Since there is only one path now, the "path not found" error is unambiguous.